system-variables.md

```diff
@@ -1522,3 +1522,12 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_enable_paging`
```

Comment:
Could you please move this sysvar to L641, just above `tidb_enable_parallel_apply`? 
It'll be easier for users to locate it if we sort all sysvars in alphabetical order.